src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -205,7 +206,8 @@ public static JSONObject parseVideo(final BufferedReader br) throws IOException
     }
     
     private static long parseNumber(String n) throws NumberFormatException {
-        return Long.parseLong(numberfix.matcher(n).replaceAll(""));
+        String withoutSpaces = CharMatcher.WHITESPACE.removeFrom(n);
```

Comment:
But this implementation will break because comma in numbers will not be considered. A better solution would be to include space in the regex used for replacement.